### PR TITLE
Internal Bleeding Rework

### PR DESCRIPTION
**Pull request by Dave-TH**
This PR proposes a slight rework to how internal bleeding manifests. Currently internal bleeding is represented by the mob occasionally dripping blood on the floor exact how external bleeding does, which isn't a great way to show the player they're bleeding *internally*. This rework makes internal bleeding manifest in the following ways: There's a 5% chance per internally bleeding wound that the mob will cough up blood on a bleed cycle. If the mob has more than one wound with internal bleeding there's also a 2.5% chance per wound that the mob will vomit blood.

Basically: The more severe the internal bleeding, the more frequently you'll be coughing (Or maybe vomiting) blood!

Note: This PR *does not* change the rate of blood loss for internal bleeding or anything similar, the differences are entirely cosmetic.

Fix(es?), unsure how many the maints want to charge me for this: #10160, #10210, #9922

:cl:
tweak: Internal bleeding is now represented by coughing (And occasionally vomiting) blood! Nifty!
/:cl:

### CODE REVIEW

**Comment by variableundefined** [comment]
I like the feature. Counting it as free.

File: code/modules/surgery/organs/blood.dm [hunk 1]
```diff
@@ -108,6 +108,26 @@
...
 
+/mob/living/carbon/proc/bleed_internal(amt) // Return 1 if we've coughed blood up, 2 if we're vomited it.
+	if(blood_volume)
+		blood_volume = max(blood_volume - amt, 0)
+		if (prob(10 * amt)) // +5% chance per internal bleeding site that we'll cough up blood on a given tick.
+			custom_emote(1, "coughs up blood!")
+			add_splatter_floor(loc, 1)
+			return 1
+		else if (amt >= 1 && prob(10 * amt)) // +5% chance per internal bleeding site that we'll cough up blood on a given tick. Must be bleeding internally in more than one place to have a chance at this.
+			vomit(0, 1)
```
Comment (KasparoVy) [inline code/modules/surgery/organs/blood.dm]: Bloooooooood 👹👌

**Comment by Dave-TH** [comment]
Update: Halved the rate of blood vomiting, after tests it was too frequent and annoying.
